why-quarto.qmd

```diff
@@ -1,51 +1,202 @@
-taken over as the tool to use when working with R and data science. We
-will use this tool because it aligns with our [values and philosophies](https://seedcase-project.org/about.html#principles)
-for multiple reasons.
+taken over as the tool to use when working with R and data science.
 
-1.  It is open source.
-2.  The documentation is phenomenal and very beginner friendly.
-3.  It is sponsored by an organization ([Posit](https://posit.co/)) with
+::: columns
+::: column
+#### Benefits
+
+-   The documentation is fantastic and very friendly to beginners.
+-   It is sponsored by an organization ([Posit](https://posit.co/)) with
     an established history of supporting and contributing to open source
     projects.
-4.  The software design is extremely well developed and structured,
+-   The software design is extremely well developed and structured,
     which makes it easier to use compared to other alternatives.
-5.  It has multi-language support (like R and Python), but is itself
+-   It has multi-language support (like R and Python), but is itself
     language-agnostic.
-6.  Because it is integrated into RStudio, many data scientists and
-    researchers who use R will quickly become familiar with it.
-7.  It has support for integration with other applications, like [VS
+-   It has support for integration with other applications, like [VS
     Code](https://code.visualstudio.com/), which is also used by many
     software developers and data scientists.
-8.  It is built on top of [Pandoc Markdown](https://pandoc.org/), which
+-   It is built on top of [Pandoc Markdown](https://pandoc.org/), which
     the de facto standard for writing Markdown, with strong historical
     use and community support.
+-   It has a large number of built-in functionality and features, while
+    still being easy to use.
+-   It is very easy to customize and extend the website to fit ones
+    needs, with themes and plugins that are easy to develop and install.
+:::
+
+::: column
+#### Drawbacks
+
+-   It is a relatively new tool, so there will be some bugs and issues
+    that come up along the way.
+-   Isn't as fast to build as other tools, like mdBook.
+-   While not necessary for most purposes, to effectively develop
+    features in an extension requires some knowledge of
+    [Lua](https://www.lua.org).
+:::
+:::
+
+## Decision outcome
+
+::: content-hidden
+What decision was made, use the form "We decided on CHOICE because of
+REASONS."
+:::
 
-For these reasons, we write our documentation that includes the website
-with Quarto.
+We have decided to go with Quarto as this seems to be the most
+accessible, as well as being a tool that there is already support for
+internally in the team. It has the most functionality when it comes to
+features, theming, and customization, while still being very easy to use
+if you have very basic knowledge of Pandoc Markdown, building websites,
+and file paths.
```

Comment:
Is the template header "Consequences" optional or is it missing here?

Reply:
I think its optional if there aren't any major consequences.